src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizer_user_pass_function/iot_custom_authorizer_user_pass.py

```diff
@@ -0,0 +1,45 @@
+import json
+import base64
+import os
+
+def handler(event, __):
+    print(f"### handler: {event}")
+    token = event["token"].lower()
+
+    mqtt_user = event["protocolData"]["mqtt"]["username"]
+    ios_token = os.environ["custom_auth_user_pass_username"]
```

Comment:
Is this more appropriately named `expected username token` or similar?

Reply:
Updated.